sdk/src/types/block/protocol.rs

```diff
@@ -180,7 +184,7 @@ impl ProtocolParameters {
 #[getset(get_copy = "pub")]
 pub struct WorkScoreStructure {
     /// Modifier for network traffic per byte.
-    data_kilobyte: u32,
+    data_byte: u32,
```

Comment:
is this byte or kibibyte? in TIP it's kibibyte, no?

Reply:
It's byte, it'll be changed in the TIP